strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PypiWheelArtifactCoordinates.java

```diff
@@ -0,0 +1,239 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import javax.persistence.Entity;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.Map;
+
+import org.carlspring.strongbox.util.PypiWheelArtifactCoordinatesUtils;
+import org.semver.Version;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Represents {@link ArtifactCoordinates} for PyPi repository
+ * 
+ * Proper path for this coordinates is in the format of: 
+ * {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl.
+ * Example: distribution-1.0-1-py27-none-any.whl
+ * 
+ * @author alecg956
+ */
+@Entity
+@SuppressWarnings("serial")
+@XmlRootElement(name = "PypiWheelArtifactCoordinates")
+@XmlAccessorType(XmlAccessType.NONE)
+@ArtifactCoordinatesLayout(name = PypiWheelArtifactCoordinates.LAYOUT_NAME, alias = PypiWheelArtifactCoordinates.LAYOUT_ALIAS)
+public class PypiWheelArtifactCoordinates
+    extends AbstractArtifactCoordinates<PypiWheelArtifactCoordinates, Version>
+{
+
+    public static final String LAYOUT_NAME = "PyPi";
+    public static final String LAYOUT_ALIAS = "pypi";
+
+    public static final String DISTRIBUTION = "distribution";
```

Comment:
Would you mind changind these variables as follows:
```
public static final String BUILD = "build";

public static final String LANGUAGE_IMPLEMENTATION_VERSION = "languageImplementationVersion";

private static final String ABI = "abi";

private static final String PLATFORM = "platform";
```

Cosmetic: Could you please keep an empty line between fields and constants?

Reply:
Should be resolved.